### ISSUE TITLE
Rework symbolsFromString

### ISSUE THREAD

**Issue by balidani** [balidani/emoji#12]
symbolsFromString does not work for some emoji (such as 🛍)

`'🛍'.length` is 2, the second bit being an emoji variation selector (https://unicode-explorer.com/c/FE0F.)

We should revert to referring to symbols by their class name. When we import symbols in `Catalog::updateSymbols`, we could extract the name and add it to some mapping.

**Comment by dave-gray101** [comment]
My thought process of referring to emoji by 'literal emoji' rather than <Class>.name was to allow symbols to reference thematically related symbols either through groups or specific name, even if those emoji were overridden by another, unknown js file without importing it.

An earlier draft of updateSymbols did just that - mapping from both the name and the symbol - but I failed to see a reason to use the name based version since javascript allows emoji in string literals?

I do agree that the multicharacter parsing thing is a bug - so I'll fix that ASAP since I just used a naive method of getting the emoji from the string.
